cmd/collector/app/zipkin/json.go

```diff
@@ -37,7 +37,7 @@ type annotation struct {
 type binaryAnnotation struct {
 	Endpoint endpoint `json:"endpoint"`
 	Key      string   `json:"key"`
-	Value    string   `json:"value"`
+	Value    json.Number   `json:"value"`
```

Comment:
This feels hacky, I am using this https://stackoverflow.com/a/24480906/4158442. 

At the moment we also do not parse type.

Reply:
@yurishkuro what about using `json.RawMessage` instead? it's basically `[]byte`

Reply:
it's harder to parse, up to you. Does zipkin allow other unquoted values like `true`?

Reply:
`json.RawMessage` keeps `"`  there `"value": "str"` is `"str"`

Reply:
we should also base64 decode if the type is bytes. I will add type deserialization and some logic for it. Do not merge yet